lib/lita/handlers/reviewme.rb

```diff
@@ -123,6 +123,14 @@ def github_access_token
       def chat_mention(reviewer, url)
         "#{reviewer}: :eyes: #{url}"
       end
+
+      def get_room(response)
+        response.message.source.room
```

Comment:
It looks like `room_object` is delegated to the source (https://github.com/litaio/lita/blob/94ad234c9737d658d1b0d8d7829530e4b378dc62/lib/lita/message.rb#L30). I wonder if it'd be _slightly better_ do so something like.

```ruby
response.message.room_object

# and elsewhere
room.name
```